### ISSUE TITLE
mvn clean package got stuck

### ISSUE THREAD

**Issue by malikhan5049** [jhipster/generator-jhipster#1724]
Hi,

I am doing mvn clean package my project and on concsole it got stuck after printing
Deleting C:\asdf\asdf\project\node_modules (includes = [], excludes = [])

**Comment by malikhan5049** [comment]
this is because mvn-clean-plugin in pom.xml is configured with <configuration><fileset><directory>node_modules<><><>, anyway do anyone have any idea how to do mvn clean package in some faster way??

**Comment by gmarziou** [comment]
Let me rephrase what I understood; you find that the clean configuration takes too long because it deletes the node_modules folder.

And I would add that as you're on Windows this could cause problems because some files could be locked either by your IDE or some other program.

Then just edit your pom.xml and remove node_modules folder from the mvn-clean-plugin configuration.

**Comment by jdubois** [comment]
Yes, I don't think we could/should change this. We delete this folder to get again the latest version of everything, as well as re-download stuff (as there can also be download issues, so you do a "mvn clean" to have something clean to start up again).
